frontend/src/containers/Header/index.tsx

```diff
@@ -1,25 +1,29 @@
 import styles from './styles.module.sass';
-
 import React, { FunctionComponent } from 'react';
 import { connect } from 'react-redux';
 import UserAvatar from '../../components/UserLogo';
 import UserPopUp from '../../components/UserPopUp';
 import SearchInput from '../../components/SearchInput';
 import { IAppState } from '../../common/models/store';
 import { IUserState } from '../../reducers/user';
-import logo from '../../img/chatitoTemp.png';
+import logo from '../../img/ChatitoTemp.png';
+import EditProfile from '../EditProfile';
+import { modal as modalRoutine, IModalRoutine } from '../../routines/modal';
+import { ModalTypes } from '../../common/enums/ModalTypes';
 
 interface IProps {
   user: IUserState;
+  modal: ({ modalType, show }: IModalRoutine) => void;
```

Comment:
```suggestion
  showModal: ({ modalType, show }: IModalRoutine) => void;
```

modal doesn't look like a method